framework/moose.mk

```diff
@@ -376,6 +376,12 @@ endif
 wasp_submodule_status:
 	@if [ x$(wasp_submodule_message) != "x" ]; then printf $(wasp_submodule_message); exit 1; fi
 
+# pre-make for checking current dependency versions and showing useful warnings
+# if things like conda packages are out of date
+prebuild:
+	@-python $(FRAMEWORK_DIR)/../scripts/premake.py
```

Comment:
```suggestion
	@-python3 $(FRAMEWORK_DIR)/../scripts/premake.py
```

?

Reply:
Something like this maybe: https://github.com/idaholab/moose/blob/2fd248e338982ab5b2dc53548af01210c5aa7ace/framework/moose.mk#L113